src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
+$cell-size: 100px;
+$padding: 10px;
+$gap: 1px;
+$days-of-week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+$background-color: #eee;
+$hover-color: #ffbfcb;
+$width: (($cell-size + $gap) * 7) - $gap;
+
 body {
   margin: 0;
+  height: 100vh;
+  justify-content: center;
+  display: flex;
+  align-items: center;
+}
+
+.calendar {
+  width: $width;
+  padding: $padding;
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    width: $cell-size;
+    height: $cell-size;
+    background-color: $background-color;
+    border: 1px solid black;
+    position: relative;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+        position: absolute;
```

Comment:
You don't need to write this classes inside loop. Use them in .calendar__day instead. Also you can remove position absolute at all.